spec/Section 6 -- Execution.md

```diff
@@ -143,6 +143,84 @@ ExecuteMutation(mutation, schema, variableValues, initialValue):
     selection set.
   * Return an unordered map containing {data} and {errors}.
```

Comment:
Putting a comment here because silly GitHub won't let me comment on the lines above. Up on line 106-107 it says:

> If mutations are supported, it must also provide a mutation root object type.

That should probably say:

> If mutations or subscriptions are supported, it must also provide a mutation and subscription root object type, respectively.

Or words to that effect.